helm_deploy/hmpps-manage-adjudications-api/values.yaml

```diff
@@ -52,22 +52,6 @@ generic-service:
       DATABASE_NAME: "database_name"
       DATABASE_USERNAME: "database_username"
       DATABASE_PASSWORD: "database_password"
-    hmpps-domain-events-topic:
```

Comment:
Moved into each environment's config, will revert once we're happy to make same changes in PreProd & Prod